src/modules/invoices/services/invoices.service.ts

```
@@ -12,6 +12,7 @@ import type {
   InvoiceSummary,
   SelectInvoiceLineItem,
 } from '@/modules/invoices/types/invoices.types';
+import { invoiceValidations } from '@/modules/invoices/schemas/invoices.validation';
```

ESLint

[error] `@/modules/invoices/schemas/invoices.validation` import should occur before import of `@/modules/invoices/services/invoice-client-resolver.service`

(import/order)